Server/src/controllers/authenticationController.ts

```diff
@@ -147,7 +146,7 @@ export class AuthenticationController {
     async createFetchUserDetailsRequest(request, response): Promise<Response> {
         let serviceResponse: IResponse;
         try {
-            let userEmail: any = request.body.email;
+            let userEmail: any = request.query.email;
```

Comment:
Why did we change this back to query?

Reply:
because its a get request, maybe i changed it to body by accident, but the idea is that we are querying the user details by email, so the email comes from the query, and also get requests dont have body i think